x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_header.test.tsx

```diff
@@ -13,57 +13,64 @@ import { ResultHeader } from './result_header';
 describe('ResultHeader', () => {
   const resultMeta = {
     id: '1',
-    scopedId: '1',
     score: 100,
     engine: 'my-engine',
   };
 
   it('renders', () => {
-    const wrapper = shallow(<ResultHeader showScore={false} resultMeta={resultMeta} />);
+    const wrapper = shallow(
+      <ResultHeader showScore={false} resultMeta={resultMeta} isMetaEngine={false} />
+    );
     expect(wrapper.isEmptyRender()).toBe(false);
   });
 
   it('always renders an id', () => {
-    const wrapper = shallow(<ResultHeader showScore={false} resultMeta={resultMeta} />);
+    const wrapper = shallow(
+      <ResultHeader showScore={false} resultMeta={resultMeta} isMetaEngine={false} />
+    );
     expect(wrapper.find('[data-test-subj="ResultId"]').prop('value')).toEqual('1');
   });
 
   describe('score', () => {
     it('renders score if showScore is true ', () => {
-      const wrapper = shallow(<ResultHeader showScore={true} resultMeta={resultMeta} />);
+      const wrapper = shallow(
+        <ResultHeader showScore={true} resultMeta={resultMeta} isMetaEngine={false} />
+      );
       expect(wrapper.find('[data-test-subj="ResultScore"]').prop('value')).toEqual(100);
     });
 
     it('does not render score if showScore is false', () => {
-      const wrapper = shallow(<ResultHeader showScore={false} resultMeta={resultMeta} />);
+      const wrapper = shallow(
+        <ResultHeader showScore={false} resultMeta={resultMeta} isMetaEngine={false} />
+      );
```

Comment:
[optional, not a change request] one test technique that could possibly making reading the component props a little easier to read is to declare a set of default props at the start of the test block, e.g.

```js
const props = {
  resultMeta,
  showScore: false,
  isMetaEngine: false,
};
```

and then for tests you could do something like:
```jsx
const wrapper = shallow(<ResultHeader {...props} showScore={false} />);
```
which isn't a huge deal but essentially helps highlight which prop we're actually examining for readability (similar to what we do for logic files)